canonicalwebteam/blog/common_view_logic.py

```diff
@@ -5,10 +5,10 @@
 group_cache = {}
 
 
-def transform_index_article(article):
+def get_complete_article(article, group_slug=None):
```

Comment:
The arguments name is `group_slug` but you are effectively passing the group id when you call the function. Might be worth renaming so it doesn't get confusing.